backend/plugins/authorization.js

```diff
@@ -65,7 +66,8 @@ export function PlotlyOAuth() {
                       res.setCookie('db-connector-auth-token',
                                     dbConnectorAccessToken, {
                                         'maxAge': getAccessTokenExpiry(),
-                                        'path': '/'
+                                        'path': getSetting('DB_CONNECTOR_URL'),
+                                        'secure': getSetting('PLOTLY_API_SSL_ENABLED')
```

Comment:
I think that we need a separate setting for this. Something like `CONNECTOR_SSL_ENABLED` or even just `SSL_ENABLED`. 

If someone is running this on their own servers against plot.ly, then they may or may not have SSL enabled even though https://plot.ly is.